### PR TITLE
Spec all projects

### PR DESCRIPTION
**Pull request by adsantos**
Depending on the result of a meeting today, this PR suggests a change to the test data structure.

### CODE REVIEW

**Comment by adsantos** [comment]
In the meeting, it was decided that this filter should work as it is represented in this PR.

**Comment by adsantos** [comment]
Ready for review

File: spec_all_projects.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                            "date": null,
+                            "project" : null,
+                            "items": [3,4,5,6,7],
+                            "isReorderable": true,
+                            "labels": null
+                        }
+                    ]
+                },
+                {
+                    "id": "1",
```
Comment (Garee) [inline spec_all_projects.json]: Test `id` needs to be incremented.

File: spec_all_projects.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
```
Comment (Garee) [inline spec_all_projects.json]: Test `id` needs to be incremented.

File: spec_all_projects.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
```
Comment (Garee) [inline spec_all_projects.json]: Test `id` needs to be incremented.

**Comment by adsantos** [comment]
Test ids incremented.

**Comment by adsantos** [comment]
Just in case it's forgotten, it's ready for review again.

**Pull request merged**